### DISCUSSION TITLE
How to install Stylus & setup CSS

### DISCUSSION THREAD

**Discussion by Docfxit** [openstyles/stylus#1827] [General]
Are there some instructions on how to setup Stylus and install CSS someplace?
I have installed the Stylus extension in Firefox ver. 131.0
I have created CSS code and set it up here:
moz-extension://810b3217-250d-4eeb-9e8a-f0bb39cfdc08/edit.html?id=2
I don't know how to point Firefox to the CSS code I setup.
I have downloaded the Stylus zip file but I don't know how to install it in Windows 10.
The CSS code is designed to move the controls in Youtube from the bottom of the page to the top.

**Comment by Mottie** [comment]
We have some [wiki page documents](https://github.com/openstyles/stylus/wiki/Writing-styles). Hopefully that will help.
